src/services/userTransform.ts

```diff
@@ -87,6 +88,11 @@ export class UserTransformService {
           } as ProcessorTransformationResponse);
           return transformedEvents;
         }
+        stats.counter('user_transform_input_events', events.length, { workspaceId });
+        logger.info('user_transform_input_events', {
+          inCount: events.length,
+          ...transformationTags,
+        });
```

Comment:
Do we wanna log it in future in the service ?
I guess the current logging level is WARN ?

Reply:
lets change log_level for ut services. wdyt ?

Reply:
No, let's keep it at Warn only. Sometimes we can make mistakes and log a high frequency on in the INFO.